mlflow/models/evaluation/base.py

```diff
@@ -456,11 +485,121 @@ def _start_run_or_reuse_active_run(run_id):
     else:
         if run_id and active_run.info.run_id != run_id:
             raise ValueError(
-                "An active run exists, you cannot specify another run_id when " "evaluating."
+                "An active run exists, you cannot specify another run_id when evaluating."
             )
         yield active_run.info.run_id
 
 
+def _normalize_evaluators_and_evaluator_config_args(
+        evaluators,
+        evaluator_config,
+):
+    from mlflow.models.evaluation.evaluator_registry import _model_evaluation_registry
+
+    def check_nesting_config_dict(_evaluator_name_list, _evaluator_name_to_conf_map):
+        return isinstance(_evaluator_name_to_conf_map, dict) and \
+               all(k in _evaluator_name_list and isinstance(v, dict)
+                   for k, v in _evaluator_name_to_conf_map.items())
+
+    if evaluators is None:
+        evaluator_name_list = list(_model_evaluation_registry._registry.keys())
+        if len(evaluator_name_list) > 1:
+            print(f'Hint: Multiple registered evaluators are found {evaluator_name_list} and '
+                  'they will all be used in evaluation. If you want to evaluate with one '
+                  'evaluator, specify the `evaluator` argument and (optional) specify the '
+                  '`evaluator_config` argument.')
+        if evaluator_config is not None:
+            conf_dict_value_error = ValueError(
+                "If `evaluators` argument is None, all registered evaluators will be used, "
+                "if only default evaluator available, the `evaluator_config` argument can be "
+                "config dict for default evaluator, otherwise the `evaluator_config` argument "
+                "must be a dict contains mapping from evaluator name to individual "
+                "evaluator config dict."
```

Comment:
```suggestion
                "If the `evaluators` argument is None, all available evaluators will be used. "
                "If only the default evaluator is available, the `evaluator_config` argument is "
                "interpreted as the config dictionary for the default evaluator. Otherwise, the `evaluator_config` argument "
                "must be a dictionary mapping each evaluator's name to its own evaluator config dictionary."
```